config.sh

```diff
@@ -1,16 +1,24 @@
 #!/bin/bash
 
 ## CONFIG SCRIPT
-# This script takes two command line arguments 
+# This script takes four command line arguments 
 # the first is the directory that we should be running this entire program in (/hdd for example)
 # the second is the link to the kernel that we want to run the pop paths on
+# the third is the kernel number to scrape CVEs for
+# the fourth is the directory to store the patch files in
 # rerun this script with new arguments to overwrite the current configuration
 
-if [[ $# -eq 2 ]]
-then
-    echo $1 > ./directory.txt
+if [[ $# -eq 4 ]]; then
+  # Define the filename for the configuration file
+  CONFIG_FILE="vars.env"
 
-    echo $2 > ./kernel_link.txt 
+  # Write the environment variables to the configuration file
+  echo "export DIRECTORY=$1" >> $CONFIG_FILE
+  echo "export KERNEL_LINK=$2" >> $CONFIG_FILE
+  echo "export KERNEL_NUMBER=$3" >> $CONFIG_FILE
+  echo "export STORE_PATCH_DIR=$4" >> $CONFIG_FILE
```

Comment:
i dont think we need to let the user decide where the patch files are, there sould just be a folder were all intermediate files are